Build.PL

```diff
@@ -24,6 +24,7 @@ my $builder = Module::Build->new(
         'JSON::XS'           => 0,
         'MIME::Lite'         => 0,
         'Net::SMTP::SSL'     => 0,
+        'Text::CSV'          => 0,

```

Comment:
Definitely agree here.

I'm not sure how to update the test.  I want to add SKIP clause, but since t/engine.t runs in a loop it has no knowledge of what it's running.  I could inspect the test name and then skip if it's the parse_csv test and Text::CSV isn't installed, but that's a hack.  Still, I can't think of any other alternatives right now.

How would you like me to handle the test?